translib/transformer/portchannel_openconfig_test.go

```diff
@@ -0,0 +1,299 @@
+////////////////////////////////////////////////////////////////////////////////
+//                                                                            //
+//  Copyright 2024 Dell, Inc.                                                 //
+//                                                                            //
+//  Licensed under the Apache License, Version 2.0 (the "License");           //
+//  you may not use this file except in compliance with the License.          //
+//  You may obtain a copy of the License at                                   //
+//                                                                            //
+//     http://www.apache.org/licenses/LICENSE-2.0                             //
+//                                                                            //
+//  Unless required by applicable law or agreed to in writing, software       //
+//  distributed under the License is distributed on an "AS IS" BASIS,         //
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  //
+//  See the License for the specific language governing permissions and       //
+//  limitations under the License.                                            //
+//                                                                            //
+////////////////////////////////////////////////////////////////////////////////
+
+//go:build testapp
+// +build testapp
+
+package transformer_test
+
+import (
+	"github.com/Azure/sonic-mgmt-common/translib/tlerr"
+	"testing"
+	"time"
+)
+
+func Test_openconfig_portchannel(t *testing.T) {
+	var url, url_input_body_json string
+
+	t.Log("\n\n+++++++++++++ CONFIGURING PORTCHANNEL ++++++++++++")
+
+	t.Log("\n\n--- PUT to Create PortChannel 111 ---")
```

Comment:
Please add the subsequent GET test to ensure the expected results read after PUT/PATCH/DELETE/POST.

Reply:
Thank you! Addressed by adding test after min-links delete and also after portchannel creation to make sure creation succeeded. 
Please let me know if there is anything else to change!